src/js/options.js

```diff
@@ -12,7 +12,7 @@
 	object, to make sure all values can be correctly looped elsewhere. */
 
 	var defaultFormatter = { 'to': function( value ){
-		return value.toFixed(2);
+		return value !== undefined && value.toFixed(2);
```

Comment:
Sorry, this was in a previous PR, please do not comment here, I am gonna close this one